### PR TITLE
[Enhancement] Add support for highlight-symbol & highlight-thing

### PR DESCRIPTION
**Pull request by ianpan870102**
This PR fixes https://github.com/bbatsov/zenburn-emacs/issues/347

For detailed screenshots, please kindly refer to the issue with the link above.

### CODE REVIEW

**Comment by ianpan870102** [comment]
@tarsius @bbatsov friendly pings :-) What are your thoughts on havin' this feature?

**Comment by tarsius** [comment]
I don't know as I don't currently use this theme.

**Comment by ianpan870102** [comment]
@bbatsov Friendly ping.

**Comment by bbatsov** [comment]
Fine by me. Don't forget to update the changelog.

**Comment by ianpan870102** [comment]
> Don't forget to update the changelog.

Hi @bbatsov, thanks for getting back. I'd love to update the changelog, but I don't exactly know how -- should I simply add "Add support for highlight-symbol & highlight-thing" under **New features** in **CHANGELOG.md**?

**Comment by bbatsov** [comment]
Yeah, exactly. Also add some link to #347 there.

**Comment by ianpan870102** [comment]
@bbatsov Just did both :) waiting for final review!

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 * Add cider-fringe-good-face
 * Add realgud settings
 * Customize company-quickhelp colors
+* Support for highlight-symbol and highlight-thing (#347)
```
Comment (bbatsov) [inline CHANGELOG.md:25]: Please, rebase on top of `master` and put this in the `master` section, as 2.6 was released a while ago.

Comment (ianpan870102) [reply CHANGELOG.md:33]: Just did so, let me know if there's anything else.

**Comment by bbatsov** [comment]
I still see the same diff on GitHub.

**Comment by ianpan870102** [comment]
Yes, I believe you should be seeing the same diff for my particular commit. However, since I rebased my changes onto your latest master which I fetched remotely, the **changelog** file should now be in sync with your latest changes of **changelog** -- that was what you wanted right?

**Comment by bbatsov** [comment]
I'll just fix that myself. It's easier that way. :-) Thanks!

**Pull request merged**